ui/lib/kv/addon/components/kv-secret-form.js

```diff
@@ -37,6 +41,14 @@ export default class KvSecretForm extends Component {
     return KVObject.create({ content: [{ name: '', value: '' }] }).toJSONString(true);
   }
 
+  get showAlert() {
+    const { metadata, previousVersion } = this.args;
+    if (!metadata?.currentVersion || !previousVersion) return false;
+    if (!this.args.secret.isNew) return false; // prevents alert from flashing after save before route transition
```

Comment:
Could these be combined?
```js
if (!metadata?.currentVersion || !previousVersion || !this.args.secret.isNew) return false;
```

Reply:
Yes! This getter was trickier than anticipated because of the transition issue, but that should work! Good suggestion